app/views/ShareListView/index.js

```diff
@@ -7,7 +7,7 @@ import ShareExtension from 'rn-extensions-share';
 import * as FileSystem from 'expo-file-system';
 import { connect } from 'react-redux';
 import * as mime from 'react-native-mime-types';
-import { isEqual, orderBy } from 'lodash';
+import { isEqual } from 'lodash';
```

Comment:
WDYT about use `isEqual` from `react-fast-compare`, since this is already installed